tests/e2e/specs/shopper/cart-checkout/account.test.js

```diff
@@ -15,7 +15,7 @@ import { visitAdminPage } from '@wordpress/e2e-test-utils';
 /**
  * Internal dependencies
  */
-import { shopper, merchant } from '../../../../utils';
+import { shopper, merchant, clickLink } from '../../../../utils';
 import { SIMPLE_PHYSICAL_PRODUCT_NAME } from '.../../../../utils/constants';
```

Comment:
Just noticed this!

Regarding my comment above, this is a much better way to put the string instead of inside the `block` object IMHO. Also, it's literally the same string as your `foundProduct`, so what do you say about using this constant instead of the `foundProduct` above?